patches/0001-Revert-chore-use-git-fork-for-aws-nitro-enclaves-cos.patch

```diff
@@ -1,16 +1,31 @@
-From d9554495b54d1f150a85f4e329412ff62728d9c7 Mon Sep 17 00:00:00 2001
-From: Peter Robinson <pbrobinson@gmail.com>
-Date: Thu, 15 Feb 2024 12:06:44 +0000
+From 01500d1b5ac04f86a286aa028f91474842be58ce Mon Sep 17 00:00:00 2001
+From: Antonio Murdaca <amurdaca@redhat.com>
+Date: Thu, 21 Nov 2024 11:12:50 +0100
 Subject: [PATCH] Revert "chore: use git fork for aws-nitro-enclaves-cose"
 
 This reverts commit 1ea09e88baee3c47cdc76902f0034d553207d515.
 ---
+ Cargo.lock              | 3 ++-
  data-formats/Cargo.toml | 2 +-
  http-wrapper/Cargo.toml | 2 +-
  3 files changed, 4 insertions(+), 3 deletions(-)
 
-Sdiff --git a/data-formats/Cargo.toml b/data-formats/Cargo.toml
-index 714fc55..6d00eb4 100644
+diff --git a/Cargo.lock b/Cargo.lock
+index abbafca..47d6075 100644
+--- a/Cargo.lock
++++ b/Cargo.lock
+@@ -191,7 +191,8 @@ checksum = "d468802bab17cbc0cc575e9b053f41e72aa36bfa6b7f55e3529ffa43161b97fa"
+ [[package]]
+ name = "aws-nitro-enclaves-cose"
+ version = "0.4.0"
+-source = "git+https://github.com/nullr0ute/aws-nitro-enclaves-cose/?rev=e3938e60d9051690569d1e4fcbe1c0c99d2fafa8#e3938e60d9051690569d1e4fcbe1c0c99d2fafa8"
++source = "registry+https://github.com/rust-lang/crates.io-index"
++checksum = "4e2fe3e862758ef5bb5d89868141ab28781d96347522b60eb6abeaf7f9acd4bc"
```

Comment:
I think we must not include `Cargo.lock` in the patch to prevent conflicts with `dependantbot`: If `dependantbot` changes the `Cargo.lock` file it's possible the patch above does not apply cleanly and it will break the build (preventing automatic crate updates).

In fact, we do not need to patch `Cargo.lock` anyway because it is not used when building the RPMs: `Cargo.lock` is removed by the `%cargo_prep` macro (except in RHEL 8, this is why a `rm -f Cargo.lock` is still there for RHEL, maybe we can also add a comment explaining it...)

Reply:
done